.github/workflows/tests.yml

```
@@ -0,0 +1,94 @@
+name: Tests
+
+# Controls when the action will run. 
+on: [push, pull_request]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

```suggestion
on: [push, pull_request, workflow_dispatch]
```

This should fix the syntax error.

---

Wow, this is bad: I got no error message, that the test could not be executed. I thought everything worked.

---

double-high-five on including workflow_dispatch in the first place. I never fail to regret omitting it from a workflow.